monkey/monkey_island/cc/ui/src/components/pages/LandingPage.tsx

```diff
@@ -1,31 +1,44 @@
 import React from 'react';
-import {Col, Row} from 'react-bootstrap';
+import {Container,Col, Row} from 'react-bootstrap';
 import {Link} from 'react-router-dom';
 import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import {faFileCode, faLightbulb} from '@fortawesome/free-solid-svg-icons';
 import '../../styles/pages/LandingPage.scss';
 import IslandHttpClient from "../IslandHttpClient";
 
+import ParticleBackground from '../ui-components/ParticleBackground';
+import Logo from "../logo/LogoComponent";
 
 const LandingPageComponent = (props) => {
 
   return (
+  <>
+    <ParticleBackground/>
     <Col sm={{offset: 3, span: 9}} md={{offset: 3, span: 9}}
```

Comment:
Incorrect layout for these: 
![image](https://user-images.githubusercontent.com/36815064/127142223-4737eb74-f51b-44d0-a6d9-ebc59a2e862c.png)